linetools/analysis/linelimits.py

```diff
@@ -0,0 +1,118 @@
+"""Module containing the LineLimits object
+"""
+from __future__ import print_function, absolute_import, division, unicode_literals
+
+import numpy as np
+import pdb
+import json
+import warnings
+
+
+from astropy import units as u
+from astropy.units import Quantity
+from astropy import constants as const
+
+ckms = const.c.to('km/s')
+
+class LineLimits(object):
+    """ An object for handling the 'limits' of a line
+
+    Parameters
+    ----------
+    zlim : `~numpy.ndarray`
+        Structured array containing all of the data
+        This can be a set of 1D spectra
+
```

Comment:
Can you explain a bit more this `zlim` input? I do not follow the idea behind it being more than a tuple (zmin, zmax).